ui.R

```diff
@@ -146,8 +146,10 @@ ui <- shinyUI(fluidPage(
                                  
                                  splitLayout(verbatimTextOutput("manpower_fte_total"),
                                              verbatimTextOutput("manpower_pte_total")),
-                                 h3("How you can acheive savings with MindShift:"),
-                                 p("MindShift offers the capability to automate manual data entry, updates, and analysis processes, facilitating a transition to a time-efficient fuel management method.\n
+
+                                 h3("How you can achieve savings with Mindshift:"),
```

Comment:
why add empty lines? @dogakantarci  ?

Reply:
Oh, I'm not really sure. I guess I did not make these changes. I suppose you tagged me mistakenly. :) @sanjmeh